internal/apps/dop/providers/issue/core/relation.go

```diff
@@ -225,8 +220,23 @@ func (i *IssueService) ValidIssueRelationTypes(ids []uint64, issueTypes []string
 	if err != nil {
 		return err
 	}
-	if len(issues) > 0 {
+	if len(issues) != len(ids) {
```

Comment:
There are two cases that can break the check:
1. duplicated issueIDs like [1, 1, 1]
2. some issue have been deleted

Although it is difficult for these cases to occur, and since you already have the full list, you can just iterate over the issue list to check.

Reply:
done